src/dialogs/CategoricalFilterDialog.ts

```diff
@@ -19,7 +19,7 @@ export default class CategoricalFilterDialog extends AFilterDialog<CategoricalCo
     const bak = <string[]>bakOri.filter || [];
     const bakMissing = bakOri.filterMissing;
     const popup = this.makePopup(`<div class="selectionTable"><table><thead><th class="selectAll"></th><th>Category</th></thead><tbody></tbody></table></div>
-        <label><input class="lu_filter_missing" type="checkbox" ${bakMissing ? 'checked="checked"' : ''}>Filter Missing</label><br>`);
+        <label><input class="lu_filter_missing" type="checkbox" ${bakMissing ? 'checked="checked"' : ''}>Filter out rows containing missing values</label><br>`);
```

Comment:
maybe a utility function would be nice which returns the label + input markup + the text `Filter out rows containing missing values`  --> then we would have it in one place